alf/networks/network.py

```diff
@@ -215,18 +215,21 @@ def __init__(self, network, n, name="NaiveParallelNetwork"):
             network (Network): the parallel network will have ``n`` copies of
                 ``network``.
             n (int): ``n`` copies of ``network``
+            name(str): a string that will be added as a prefix to the name of
```

Comment:
It's better to keep the semantics of name unchanged. The caller still have the control of the name. We make the name default to `None` to be `name + '_' network.name`

Reply:
Right, the expected behavior is more natural in this way. Changed as suggested.